scene/gui/item_list.cpp

```diff
@@ -1066,19 +1083,41 @@ void ItemList::_notification(int p_what) {
 				if (items[i].disabled)
 					modulate.a *= 0.5;
 
-				if (items[i].icon_region.has_no_area())
-					draw_texture_rect(items[i].icon, draw_rect, false, modulate);
-				else
-					draw_texture_rect_region(items[i].icon, draw_rect, items[i].icon_region, modulate);
+				if (!is_font) {
+					if (items[i].icon_region.has_no_area())
+						draw_texture_rect(items[i].icon, draw_rect, false, modulate);
+					else
+						draw_texture_rect_region(items[i].icon, draw_rect, items[i].icon_region, modulate);
+				}
 			}
 
 			if (items[i].tag_icon.is_valid()) {
 
 				draw_texture(items[i].tag_icon, items[i].rect_cache.position + base_ofs);
 			}
 
-			if (items[i].text != "") {
+			if (items[i].font_sample.is_valid()) {
+				Ref<Font> font = items[i].font_sample;
 
+				String sampled_text = "Abg";
+				Vector2 size = font->get_string_size(sampled_text);
+
+				int max_len = -1;
+
+				if (fixed_column_width)
+					max_len = fixed_column_width;
+				else if (same_column_width)
+					max_len = items[i].rect_cache.size.x;
+				else
+					max_len = size.x;
+
+				Vector2 pos = items[i].rect_cache.position + base_ofs;
+				pos.x += max_len / 2 - size.x / 2;
+				pos.y += 50;
+				font->draw(get_canvas_item(), pos, sampled_text, Color(1, 1, 1), -1);
```

Comment:
It also would be possible to just load the theme font color bere instead hardcoding white.
Than u also dont need a black bg.